src/screens/FeedScreen/Content.js

```diff
@@ -396,9 +409,12 @@ export const styles = StyleSheet.create({
     marginRight: 8
   },
   contentFeed: {
-    flex: 1,
+    height: normalize(399),
     marginTop: 0,
-    width: '100%'
+    width: '100%',
+    borderLeftWidth: 1,
+    borderRightWidth: 1,
+    borderColor: COLORS.darkGray
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `contentFeed` style has been updated with new properties. Ensure these changes align with your design requirements.
```diff
+ height: normalize(399),
+ borderLeftWidth: 1,
+ borderRightWidth: 1,
+ borderColor: COLORS.darkGray
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->